project/Dependencies.scala

```diff
@@ -23,6 +23,9 @@ object Dependencies {
     lazy val prodDeps: Seq[ModuleID] = Seq(
       "org.apache.kafka" %% "kafka" % Versions.Kafka cross CrossVersion.for3Use2_13
     )
+    lazy val testDeps: Seq[ModuleID] = Seq(
```

Comment:
:information_source: The library is not pulled transitively anymore but we do use it in our tests.